### ISSUE TITLE
Multi-class Classification (`02_pytorch_classification.ipynb`)

### ISSUE THREAD

**Issue by sascharo** [mrdbourke/pytorch-deep-learning#741]
The multi-class model coded in section 93 does a surprisingly good job even without any training, and even better without `nn.ReLU`. No matter the seed used for initialization, at epoch 0, the test accuracy is always above 70%. Why is that? I can't get it down to about 50%. I observed that in the notebook I coded along, but it's the same in the notebook from your repository.

**Comment by nickyreinert** [comment]
It must be something with the code, I suggest to do a clean copy to your own notebook, see my example. For the sake of explanation, with the flow of Daniels video, the notebook is pretty good, but if you start experimenting, you quickly get lost in all those cells.

https://colab.research.google.com/drive/1otkQBQjF8pZetWJfesZaPd1dcOLPjlu2?usp=sharing

When I decrease the learning rate, I can clearly see how the accuracy goes up from below 20 to 100%:

![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/18428490/df2cd343-13a4-4ab9-b549-58f402ca47d7)


What you also could try is to make the data set more difficult. Increase the center_std to 4 or so. Now you clearly can see how the model is struggeling:

![image](https://github.com/mrdbourke/pytorch-deep-learning/assets/18428490/d7d6eb9f-c3ab-4483-8a23-e7574d8269f7)
